src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -424,6 +424,8 @@ private async Task<EntryComponent> ExecuteRequests(Hl7.Fhir.Model.Bundle respons
                     {
                         HttpContext httpContext = request.HttpContext;
 
+                        string path = httpContext.Request.Path.Value;
```

Comment:
nit: one line of change is left in this file.

Reply:
Fixed.